manifests/config/exporter/exporter.yaml

```diff
@@ -85,8 +86,10 @@ spec:
         volumeMounts:
         - mountPath: /lib/modules
           name: lib-modules
+          readOnly: true
```

Comment:
Why remove this? Enabling only read only is more secure...

Reply:
Doesn't the change only enables `readOnly` ?

Reply:
Exactly, I enabled this for as many mounts as possible to improve security :)

Reply:
right, this seems add the `readonly` :)